python/pyarrow/_flight.pyx

```diff
@@ -1106,13 +1114,15 @@ cdef class FlightClient(_Weakrefable):
 
     @classmethod
     def connect(cls, location, tls_root_certs=None, cert_chain=None,
-                private_key=None, override_hostname=None):
+                private_key=None, override_hostname=None,
+                disable_server_verify=None):
```

Comment:
Make this consistent? verify -> verification

Reply:
A few lines lower there's a line which is just disable_server_verification=disable_server_verification which exceeds the line limit. Do you know if there's a way around this?

Reply:
I'll push a change with alternative formatting that fixes this issue